module-system/sov-state/src/scratchpad.rs

```diff
@@ -40,90 +40,75 @@ impl<S: Storage> Debug for RevertableDelta<S> {
     }
 }
 
-/// A read-write set which can be committed as a unit
-enum ReadWriteSet<S: Storage> {
-    Standard(Delta<S>),
-    Revertable(RevertableDelta<S>),
+/// This structure stores the read-write set, which will be persisted in the database at the
```

Comment:
Could you please extend purpose of these structures, in a way that you did in issue description.
Like when each of those should be used, etc